Add Solr config update tip, fixes #15

gitressa opened the pull request:
## The Issue
Fixes 
* #15.

## How This PR Solves The Issue

Adds documentation on how to fix outdated Solr config files message.

Also updates the "Alternate Core Name" section, the first step (`1. Remove the #ddev-generated at the top of the file.`) isn't necessary, it seems?

## Manual Testing Instructions

## Automated Testing Overview

<!-- Please describe the tests introduced by this PR, or explain why no tests are needed. -->

## Related Issue Link(s)

## Release/Deployment Notes

<!-- Does this affect anything else or have ramifications for other code? Does anything have to be done on deployment? -->

gitressa committed:
Add Solr config update tip

rfay commented on README.md:
````diff
@@ -49,10 +57,8 @@ services:
     - SOLR_CORENAME=somecorename
 ```
-
-1. Remove the #ddev-generated at the top of the file.
````
Why did you remove the first item here?

rfay replied:
Ah, I didn't see the context `If you want to use a core name other than the default "dev", add a .ddev/docker-compose.solr-env.yaml with these contents, using the core name you want to use:`, thought it was referring to docker-compose.solr.yaml.

rfay reviewed (changes requested):
It all looks great, just the one question about why you removed #ddev-generated notice.

rfay commented:
If you don't remove the `#ddev-generated` then the file will be recreated with the default file on `ddev restart`.

gitressa commented:
I removed it because after creating the file _.ddev/docker-compose.solr-env.yaml_, adding this:

```
services:
  solr:
    environment:
    - SOLR_CORENAME=somecorename
```

and restarting DDEV. At no point do I see a line saying `#ddev-generated`, so I removed that first line ...

Or is that first line about something else?

gitressa commented:
Great, thanks!

rfay commented:
Thanks for the contribution and for keeping your eye on this repo. You know and use solr far more than I do, and it needs knowledgable folks following it.
